src/app/app-routing.module.ts

```diff
@@ -46,6 +47,11 @@ export const routes: Routes = [
         path: ProjectLinks.URL_ADMIN_DASHBOARD,
         component: AdminDashboardComponent,
     },
+    {
+        canActivate: [AdminGuard],
```

Comment:
The ticket is not saying anything that this new page should be limited to admins

Reply:
I will clarify the requirements